modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -1,20 +1,48 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
+// * post_installation_configuration/cluster-tasks.adoc
 
 [id="binding-infra-node-workloads-using-taints-tolerations_{context}"]
 = Binding infrastructure node workloads using taints and tolerations
 
 If you have an infra node that has the `infra` and `worker` roles assigned, you must configure the node so that user workloads are not assigned to it.
 
+[IMPORTANT]
+====
+It is recommended to preserve the dual `infra,worker` label that is created for infra nodes and use taints and tolerations to manage nodes user workloads are scheduled on. If you remove the `worker` label from the node, you must create a custom pool to manage it; a node with a label other than `master` or `worker` is not recognized by the MCO without a custom pool. Maintaining the `worker` label allows the node to be managed by the default worker machine config pool, and the `infra` label communicates to the cluster that it does not count toward the total number of subscriptions.
```

Comment:
Should this link to custom pool documentation

Reply:
@mburke5678 correct me if I am wrong but the way our docs; work we can't link to other sections of docs effectively.

Reply:
Eric, that is correct. We are working out the details to get them back.